src/graphicswin.cpp

```diff
@@ -281,7 +281,7 @@ void GraphicsWindow::PopulateMainMenu() {
         } else if(Menu[i].cmd == Command::LOCALE) {
             Platform::MenuRef localeMenu = currentSubMenu->AddSubMenu(label);
             for(const Locale &locale : Locales()) {
-                localeMenu->AddItem(locale.displayName, [&]() {
+                localeMenu->AddItem(locale.displayName, true, [&]() {
```

Comment:
Please add the comment on each call site indicating the purpose of the boolean argument, i.e. `/*mnemonic=*/true`, as it's done elsewhere. Otherwise it's quite confusing.

Reply:
Thanks!